src/Agent/NewRelic/Agent/Core/Core.csproj

```diff
@@ -105,14 +109,15 @@
       <ILRepackInclude Include="@(PossibleRefsForILRepack)" Condition="'%(FileName)' == 'Serilog.Sinks.Debug'" />
       <ILRepackInclude Include="@(PossibleRefsForILRepack)" Condition="'%(FileName)' == 'Serilog.Sinks.EventLog'" />
       <ILRepackInclude Include="@(PossibleRefsForILRepack)" Condition="'%(FileName)' == 'Serilog.Sinks.File'" />
+      <ILRepackInclude Include="@(PossibleRefsForILRepack)" Condition="'%(FileName)' == 'System.Memory'" />
```

Comment:
Is this now required for the netstandard2.0 target? I thought it was part of .net and only required for .net framework?

Reply:
Integration tests pass after reverting.